R/rcodesign/build_tarballs.jl

```diff
@@ -24,35 +23,36 @@ if [[ "${target}" == x86_64-apple-darwin* ]]; then
     cp -ra System "/opt/${target}/${target}/sys-root/."
     popd
 fi
-cd $WORKSPACE/srcdir/PyOxidizer-apple-codesign-*/apple-codesign/
-cargo build --release
-install -Dvm 755 "../target/${rust_target}/release/rcodesign${exeext}" "${bindir}/rcodesign${exeext}"
-install_license ../LICENSE
+
+cd $WORKSPACE/srcdir/apple-platform-rs-apple-codesign-$version/
+cargo build --release --package apple-codesign --target-dir ${WORKSPACE}/tmp
+
+install_license $WORKSPACE/srcdir/apple-platform-rs-apple-codesign-$version/apple-codesign/LICENSE
+install -Dvm 755 "${WORKSPACE}/tmp/${rust_target}/release/rcodesign${exeext}" "${bindir}/rcodesign${exeext}"
 """
 
+script = replace(script, raw"$version" => version)
+
```

Comment:
```suggestion
```

Reply:
I don't see much benefit in this complication.

Reply:
Agreed. Using `*` for the version in the path is cleaner.